src/app/app.module.ts

```diff
@@ -23,7 +23,7 @@ import {
   UserService
 } from './shared';
 
-const rootRouting: ModuleWithProviders = RouterModule.forRoot([], { useHash: true });
```

Comment:
does the backend api support "hash"-less routing?

Reply:
The backend is just an API so hash based routing plays no part there

Reply:
Oh right, it's not served by the backend server 👍